src/neuroconv/datainterfaces/ecephys/plexon/plexondatainterface.py

```diff
@@ -48,6 +48,49 @@ def get_metadata(self) -> DeepDict:
         return metadata
 
 
+class Plexon2RecordingInterface(BaseRecordingExtractorInterface):
+    """
+    Primary data interface class for converting Plexon2 data.
+
+    Uses the :py:class:`~spikeinterface.extractors.Plexon2RecordingExtractor`.
+    """
+
+    display_name = "Plexon2 Recording"
+    associated_suffixes = (".pl2",)
+    info = "Interface for Plexon2 recording data."
+
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        source_schema = super().get_source_schema()
+        source_schema["properties"]["file_path"]["description"] = "Path to the .pl2 file."
+        return source_schema
+
+    def __init__(self, file_path: FilePathType, verbose: bool = True, es_key: str = "ElectricalSeries"):
+        """
+        Load and prepare data for Plexon.
+
+        Parameters
+        ----------
+        file_path : str or Path
+            Path to the .plx file.
+        verbose : bool, default: True
+            Allows verbosity.
+        es_key : str, default: "ElectricalSeries"
+        """
+        stream_id = "3"  # TODO figure out if "4" is not raw signal as well
+        super().__init__(
+            file_path=file_path,
+            verbose=verbose,
+            es_key=es_key,
+            stream_id=stream_id,
+        )
+
+    def get_metadata(self) -> DeepDict:
+        metadata = super().get_metadata()
+
+        return metadata
```

Comment:
```suggestion
```

there is no need to override this if you are not changing the metadata at all

Reply:
Yes, but I might, it looks liker there is session data in the annotations.  I just wanted to get it running quickly, I will feel this with some of the annotations available!